src/patterns/bank-details/index.md.njk

```diff
@@ -56,10 +56,6 @@ Let users choose to get paid by an alternative method.
 
 Adapt this question depending on what payment options your users need and what your service can support.
 
-### Turn off HTML5 validation
-
-You should [turn off HTML5 validation](/patterns/validation/#turn-off-html5-validation) to prevent browsers from validating the `pattern` attribute used by the bank details pattern.
-
 {{ example({group: "patterns", item: "bank-details", example: "branch", html: true, nunjucks: true, open: false, size: "xl"}) }}
```

Comment:
I think this example is still needed as it seems to relate to the 'Use a branching question if users can choose how to get paid' section.

It does however seem odd that the guidance about HTML5 validation was previously in between that guidance and this example though. Worth double checking that we're happy this still belongs.

Reply:
Just wondering, was this example originally designed to serve 2 purposes?
Edit does look fine tho 👍

Reply:
I don't believe so – [the 'branch' example uses a set of radios to ask the user how they'd to be paid](https://deploy-preview-2348--govuk-design-system-preview.netlify.app/patterns/bank-details/#use-a-branching-question-if-users-can-choose-how-to-get-paid).

It doesn't ask for any numbers, so I don't see how it would demonstrate the usage of the `pattern` attribute.